docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,43 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main features of the Metadata API include:
+
+- Extraction of a metadata from a file
+  You can [extract metadata](#extract-metadata) from multimedia files using `get()` method.
+
+
+## Prerequisites
+
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html)), the application has to request [proper storage permissions](../security/privacy-related-permissions.md).
+
+## Extract metadata
+
+1. Create a file handle using `createFileHandle()` method:
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is properly created metadata can be extracted:
+
+   ```javascript
+   console.log("Video duration is: " + fileHandle.get("DURATION") + " milliseconds");
+   console.log("Video title is: " + fileHandle.get("TITLE"));
+   ```
+
+3. For better performance, FileHandle does not release underlying resources after every metadata extraction. On first extraction, data is decoded, thanks to that next metadata extractions are faster.
```

Comment:
@mijin85cho request your opinion on the suggestion; if you do not like the suggestion, request you to provide your suggestion. 

I am suggesting changes because the sentence seems to be in colloquial language.

```suggestion
3. For better performance, the file handle does not release underlying resources after every metadata extraction. The data is decoded on the first extraction, which helps in speeding up the next metadata extractions.
```


@pkosko I have a doubt: 

Have you intentionally kept FileHandle case in line 31 as such, whereas the case differs  in line 17
 and 24.

In my suggestion, I have changed **FileHandle** to **file handle**. If the answer to above statement is **yes**, replace file handle with FileHandle then.

Reply:
>I am suggesting changes because the sentence seems to be in colloquial language.

I think @nawab-samsung 's suggest to change the tone is good. Thanks for that.